src/common/options.rs

```diff
@@ -141,6 +141,11 @@ pub struct Options {
     nwmgr_cfg: Option<Vec<PathBuf>>,
     #[clap(long, value_name = "DT_SLUG", help = "Device Type slug to change to")]
     change_dt_to: Option<String>,
+    #[clap(
```

Comment:
Move this option up beside all the other logging options to make it easier for the user to choose.

Reply:
Another option to reduce the proliferation of options is to repurpose `-l / --log-to` to accept "balenaos". Is there any danger of conflict? Don't we expect external devices always to start with `/dev` ?